deepchem/models/IRV.py

```diff
@@ -5,12 +5,26 @@
 from deepchem.models.losses import SigmoidCrossEntropy
 from tensorflow.keras.layers import Input, Layer, Activation, Concatenate, Lambda
 
+import torch
```

Comment:
You need a unit test that the new model produces the same outputs on the same inputs as the TF model when the weights are set to the same values.